### PR TITLE
show related blogs and comparison pages

### PR DESCRIPTION
**Pull request by anirudh-tantry**
Show other comparison pages at bottom of each comparison page and blogs at bottom of each blog
image
image

### CODE REVIEW

**Comment by piyushsinghania** [comment]
Font weight 700 looks very bold. I feel it can be reduced a bit to something like 600 or 500 which we are using for other page heading.

image
image

**Comment by piyushsinghania** [comment]
Font weight and font size can be reduced a bit (comparable to other headings/sub-headings in the same page)
image

File: pages/blog/_slug.vue [hunk 1]
```diff
@@ -111,7 +123,11 @@ export default {
   async asyncData({ $content, params }) {
     const article = await $content('blog', params.slug).fetch()
 
-    return { article }
+    let relatedArticles = await $content('blog').fetch()
+    relatedArticles = relatedArticles.filter(relatedArticle => article.slug !== relatedArticle.slug)
+    const randIndex = Math.floor(Math.random() * (relatedArticles.length - 2))
```
Comment (piyushsinghania) [inline pages/blog/_slug.vue:128]: Reason for doing `relatedArticles.length - 2`

File: components/RelatedArticleCard.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      :src="article.coverImg"
+      class="rounded img-fluid"
+      :alt="article.coverImgAlt"
+      :title="article.coverImgAlt"
+    />
+    <div class="d-flex flex-column align-items-start">
+      <div class="mt-2">
+        <h3 class="article__title">{{ article.title }}</h3>
+      </div>
+    </div>
```
Comment (piyushsinghania) [inline components/RelatedArticleCard.vue:13]: May be we can reduce a nesting and move mt-2 above?😛 

```
<div class="d-flex flex-column align-items-start mt-2">
        <h3 class="article__title">{{ article.title }}</h3>
</div>
```

File: pages/blog/_slug.vue (hunk 1 above)
Comment (anirudh-tantry) [reply pages/blog/_slug.vue:128]: yes relatedArticles should always return 2 articles so largest randIndex will be 1 less than last index. while slicing 2 articles it'll include the last article also

**Comment by anirudh-tantry** [comment]
> Font weight and font size can be reduced a bit (comparable to other headings/sub-headings in the same page) image

image
For related blogs this font size and class to match its theme should I reduce it?

**Pull request merged**